HDSAviz/data_processing.py

```diff
@@ -0,0 +1,197 @@
+""" Tools for reading and processing the sensitivity analysis data files."""
```

Comment:
I think you are commenting on mine about this but I'm just going to make a note here so we remember to do this. Lets touch base on how the files and data frames are formatted because looking this over I know the inputs to my function must change.